.eslintrc

```diff
@@ -6,6 +6,9 @@
     //     "eslint:recommended",
     //     "plugin:react/recommended",
     // ],
+    "extends": [
+        "eslint:recommended",
+    ],
```

Comment:
We could allow `comma-dangle` and drop out the commit that fixes the occurrences of `comma-dangle`.